### ISSUE TITLE
Populate list of required icons

### ISSUE THREAD

**Issue by alice-i-cecile** [bevyengine/bevy_editor_prototypes#44]
Populate https://github.com/orgs/bevyengine/projects/21/views/1?visibleFields=%5B%22Title%22%2C136081107%2C136081126%2C136081152%2C136081390%5D.

Ping me if you need permissions.

Possible icon sets: https://iconify.design/

Blenders list of required icons: https://developer.blender.org/docs/features/interface/icons/

**Comment by arocide** [comment]
With the Public Figma File using almost exclusively [Lucide Icons](https://lucide.dev/icons) it might make sense for us to use that the projects placeholder icons at this stage, if so I might be able to start populating the project with a set?

**Comment by alice-i-cecile** [comment]
Yep that's fine by me, as long as we're in compliance with the license terms.

**Comment by arocide** [comment]
This should be okay, it uses an ISC License (Functionally equivalent to the MIT), speaking of Figma I'll begin the process there with icon candidates for comment before committing to the placeholders.
